packages/reactive-element/README.md

```diff
@@ -82,72 +76,14 @@ standard currently available in [TypeScript](https://www.typescriptlang.org/) or
 
 ## Installation
 
-From inside your project folder, run:
-
 ```bash
 $ npm install @lit/reactive-element
 ```
 
-To install the web components polyfills needed for older browsers:
+Or use from `lit`:
 
 ```bash
-$ npm i -D @webcomponents/webcomponentsjs
-```
-
-## Development mode
-
-`@lit/reactive-element` includes a development mode which adds additional checks that are
-reported in the console.
-
-To enable development mode, add the `development` exports condition to your node
-resolve configuration.
-
-#### @web/dev-server
-
-```js
-{
-  nodeResolve: {
-    exportConditions: ['development'],
-  }
-}
-```
-
-#### Rollup
-
-```js
-{
-  plugins: [
-    nodeResolve({
-      exportConditions: ['development'],
-    }),
-  ],
-}
-```
-
-#### Webpack
-
-> NOTE: Requires [Webpack v5](https://webpack.js.org/migrate/5/)
-
-```js
-{
-  resolve: {
-    conditionNames: ['development'],
-  }
-}
-```
-
-## Supported Browsers
-
-The last 2 versions of all modern browsers are supported, including
-Chrome, Safari, Opera, Firefox, Edge. In addition, Internet Explorer 11 is also supported.
-
-Edge and Internet Explorer 11 require the web components polyfills and the
-`polyfill-support` module included in this package.
-
-```html
-<script src="node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
-<script src="node_modules/@lit/reactive-element/polyfill-support.js"></script>
-<!-- load application code -->
+$ npm install lit
```

Comment:
Intentional to remove the dev mode stuff?

Reply:
Yeah, before we launched the RC, we had out cleaned a bunch of extra detail in the readmes to just prefer pointing to lit.dev (https://github.com/lit/lit/pull/1783/files). I think we had just missed `reactive-element` in that cleanup. `lit-element` doesn't even have a code example anymore, but I don't think we have good treatment of using `ReactiveElement` standalone on lit.dev, so seemed ok to keep that.